extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -53,10 +53,10 @@
    z-index: 2147483646;
    width: 100%;
    position: fixed;
-   top: 0;
+   background-color: white;
  }
 
- .i-amphtml-toolbar > ul {
+ .i-amphtml-toolbar > ul:first-child {
```

Comment:
The way you should ensure it is `.i-amphtml-toolbar > :not(ul:first-child) ` right (or somecss that works?)